dspace-api/src/main/java/org/dspace/app/util/DCInput.java

```diff
@@ -114,7 +114,8 @@ public DCInput(Map<String, String> fieldMap, Map<String, List<String>> listMap)
         valueLanguageList = new ArrayList();
         if (language)
         {
-            valueLanguageList = listMap.get(LanguageName);
+            languageName = fieldMap.get("language-value-pairs-name");
```

Comment:
Ah scratch this. I just saw that `DCInputsReader` is enforcing the language attribute.